drivers/slipdev/slipdev.c

```diff
@@ -67,11 +67,95 @@ static inline void slipdev_unlock(void)
     }
 }
 
-static void _slip_rx_cb(void *arg, uint8_t byte)
+static inline void _slipdev_stdio_add_to_frame(slipdev_t * dev, uint8_t byte) {
+#if IS_USED(MODULE_SLIPDEV_STDIO)
+    if (dev->config.uart == STDIO_UART_DEV) {
```

Comment:
Why is this check needed? There is no requirement for slipdev be on the same uart as `STDIO_UART_DEV` (for `stdio_uart`)